src/components/sbb-breadcrumb-group/sbb-breadcrumb-group.e2e.ts

```diff
@@ -52,125 +56,106 @@ describe('sbb-breadcrumb-group', () => {
-      await ellipsisBreadcrumb.click();
+      const changeSpy = new EventSpy('click', ellipsisButton);
+      ellipsisButton.click();
+      await waitForLitRender(ellipsisListItemElement);
       await waitForCondition(() => changeSpy.events.length === 1);
 
-      ellipsisElement = await page.find('sbb-breadcrumb-group >>> #sbb-breadcrumb-group-ellipsis');
-      ellipsisBreadcrumb = await page.find('sbb-breadcrumb-group >>> #sbb-breadcrumb-ellipsis');
-      expect(ellipsisElement).toBeNull();
-      expect(ellipsisBreadcrumb).toBeNull();
+      ellipsisListItemElement = breadcrumbGroup.shadowRoot.querySelector(
+        '#sbb-breadcrumb-group-ellipsis',
+      );
+      ellipsisButton = breadcrumbGroup.shadowRoot.querySelector('#sbb-breadcrumb-ellipsis');
+      expect(ellipsisListItemElement).to.be.null;
+      expect(ellipsisButton).to.be.null;
     });
 
     it('should expand breadcrumbs and focus correctly by keyboard', async () => {
-      // When pressing space key on ellipsis
-      const ellipsisBreadcrumb = await page.find(
-        'sbb-breadcrumb-group >>> #sbb-breadcrumb-ellipsis',
-      );
-      await ellipsisBreadcrumb.press('Space');
-      await page.waitForChanges();
+      // When pressing the space key on ellipsis button
+      ellipsisButton.focus();
+      await sendKeys({ press: 'Space' });
+      await waitForLitRender(breadcrumbGroup);
 
       // Then focus should be on first breadcrumb
-      expect(await page.evaluate(() => document.activeElement.id)).toEqual('breadcrumb-1');
+      expect(document.activeElement.id).to.be.equal('breadcrumb-1');
 
       // When blurring the focus
-      await page.evaluate(() => (document.activeElement as HTMLElement).blur());
+      (document.activeElement as HTMLElement).blur();
 
-      // Then body should be focused
-      expect(await page.evaluate(() => document.activeElement.tagName)).toEqual('BODY');
+      // Then the body should be focused
+      expect(document.activeElement.tagName).to.be.equal('BODY');
 
       // When triggering a slotChange by removing a breadcrumb
-      await page.evaluate(() => document.getElementById('breadcrumb-6').remove());
-      await page.waitForChanges();
+      document.getElementById('breadcrumb-6').remove();
+      await waitForLitRender(breadcrumbGroup);
 
-      // Then still the body should be focused
-      expect(await page.evaluate(() => document.activeElement.tagName)).toEqual('BODY');
+      // Then the body should still be focused
+      expect(document.activeElement.tagName).to.be.equal('BODY');
     });
 
     it('should remove expand button when too less breadcrumbs available', async () => {
-      let ellipsisElement = await page.find(
-        'sbb-breadcrumb-group >>> #sbb-breadcrumb-group-ellipsis',
-      );
-      let ellipsisBreadcrumb = await page.find('sbb-breadcrumb-group >>> #sbb-breadcrumb-ellipsis');
-      expect(ellipsisElement).not.toBeNull();
-      expect(ellipsisBreadcrumb).not.toBeNull();
```

Comment:
should we not keep this check to be sure that the "start" state is correct? (has ellipsis)

Reply:
It's reasonable. I have added these two lines in all the ellipsis tests, just to be sure..